docs/enroute/Brisbane Centre/TRT.md

```diff
@@ -12,6 +12,17 @@
 
 † *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies)
 
+### CPDLC
+
+The Primary Communication Method for TRT is Voice.
```

Comment:
Sorry missed this on the first go. Table on the CPDLC page has CPDLC as primary. Honestly it's probably halfway in between, so up to you which one you go